ev-dev-tools/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = ev-dev-tools
-version = 0.0.3
+version = 0.0.5
```

Comment:
It was a 0.0.4 in between and at some point I changed another part and incremented the version number again without thinking. Its just an artifact I did not rebase/edit out later because I figured it not wort the effort.

Reply:
Sure, that makes sense.